src/matrix/room/RoomSummary.js

```diff
@@ -239,6 +239,15 @@ export class SummaryData {
         return !this.name && !this.canonicalAlias && this.heroes && this.heroes.length > 0;
     }
 
+    get avatarColorId() {
+        if (this.heroes && this.heroes.length === 1) {
+            for (const hero of this.heroes) {
```

Comment:
you can just return this.heroes[0] here, as it is an array and not just a mere iterable.

Reply:
can you also add a jsdoc comment here with a description of what this property is for, please?